native/cocos/renderer/gfx-vulkan/VKInputAssembler.cpp

```diff
@@ -39,24 +39,32 @@ CCVKInputAssembler::~CCVKInputAssembler() {
     destroy();
 }
 
-void CCVKInputAssembler::doInit(const InputAssemblerInfo &info) {
-    size_t vbCount = _vertexBuffers.size();
-
+void CCVKInputAssembler::doInit([[maybe_unused]] const InputAssemblerInfo &info) {
```

Comment:
What's `[[maybe_unused]]`?

Reply:
[[mabe_unused]] is used to suppress `Clang-Tidy: Parameter 'info' is unused`.

Reply:
Good to know, it is a new feature since c++17. We should make sure it works on iOS11 and NS.

Reply:
I checked compiler support for `maybe_unused`.
MSVC: 2017 15.3+ meets our requirement(2017 15.7+)
Apple Clang: yes if c++ 17 is supported.
Clang：3.9 (Need a NS developer tools requiremens.)

I will replace it with another method.